sources/packages/web/src/services/http/StudentApi.ts

```diff
@@ -111,37 +88,31 @@ export class StudentApi extends HttpBaseClient {
   }
 
   /**
-   * API client for student search.
-   * @param appNumber
-   * @param firstName
-   * @param lastName
-   * @returns
+   * Search students based on the search criteria.
+   * @param appNumber application number.
+   * @param firstName students first name (given names).
+   * @param lastName student last name
+   * @returns searched student details.
```

Comment:
is its a list of students or details of the student

Reply:
It is a subset with firstName, lastName, birthDate. What is the point here?